pkg/reference/reference.go

```diff
@@ -36,6 +36,26 @@ const (
 	errNoValue     = "referenced field was empty (reference may not yet be ready)"
 )
 
+// NOTE(negz): There are many equivalents of FromPtrValue and ToPtrValue
+// throughout the Crossplane codebase. We duplicate them here to reduce the
+// number of packages our API types have to import to support references.
+
+// FromPtrValue adapts a string pointer field for use as a CurrentValue.
+func FromPtrValue(v *string) string {
```

Comment:
Nit: `FromPointer` and `ToPointer` maybe? I assume there won't be int64, bool etc. equivalents of these; but if that's the case I'd name them `FromStringPtr` and `ToStringPtr`.

Reply:
Yeah I imagine we probably won't be referencing any other types - the identifiers that references resolve to seem to be strings in all cases so far.